src/__tests__/ThreeD.test.ts

```diff
@@ -109,7 +109,6 @@ const nodes = [
     subtreeSize: 13,
     boundingBox: { min: [12, 3, -19], max: [18, 6, 0] },
     sectorId: 1235,
-    metadata: {},
```

Comment:
This line should not be removed? All nodes will have an empty metadata object.

Reply:
I agree :) `git commit --amend && git push --force`.